packages/specs/schema/src/components/default/requiredMapper.ts

```
@@ -40,7 +39,7 @@ function extractRequiredProps(obj: any, schema: JsonSchema, options: JsonSchemaO
 }
 
 export function requiredMapper(obj: any, schema: JsonSchema, options: JsonSchemaOptions) {
-  if (options.groups && options.groups.includes("partial")) {
+  if (options.groups && options.groups?.includes("partial")) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)